babeld/babel_interface.c

```diff
@@ -719,6 +719,7 @@ babel_interface_close_all(void)
 {
     struct vrf *vrf = vrf_lookup_by_id(VRF_DEFAULT);
     struct interface *ifp = NULL;
+    int type = ZEBRA_ROUTE_ALL;
```

Comment:
We don't need initialization here at all, because we do it inside the loop.

Reply:
Cause I am afraid of uninitialized variables, regardless compiler settings
Fixed